share/completions/scp.fish

```diff
@@ -8,7 +8,14 @@ __fish_complete_ssh scp
 # scp specific completions
 #
 
-complete -c scp -d Hostname -a "
+#
+# Hostname
+#
+complete \
+	--command     scp \
+	--description Hostname \
+	--condition   "commandline --cut-at-cursor --current-token | string match --invert '*:*'" \
+	--arguments   "
```

Comment:
I have no idea why the existing completion uses a multiline command in this manner but it is grotesque. If you're going to change this please abstract the code into a discrete function that is invoked by this completion. There is too much magic interpolation happening in this code.